### ISSUE TITLE
[Improvement]: Make Backspace slightly larger

### ISSUE THREAD

**Issue by Glitchy-Tozier** [AnySoftKeyboard/AnySoftKeyboard#1984]
Currently, the backspace-key (and the shift-key too) are a bit small. This results in users that are switching from other popular keyboards to AnySoftKeyboard to often miss the backspace-key and instead type the letter next to it.

### Proposition
Make the Backspace-Key slightly larger to make more easily accessible to new users.

I posted some pictures of other keyboard-layouts as a reference, them being SwiftKey, Google, and Simple-Keyboard.

![gboard](https://user-images.githubusercontent.com/59611881/73060746-6fcadd00-3e90-11ea-9b34-bf0d9c8214c5.jpg)
![Simple_Keyboard](https://user-images.githubusercontent.com/59611881/73060747-70637380-3e90-11ea-866b-5ce06d7b201f.png)
![SwiftKey](https://user-images.githubusercontent.com/59611881/73060748-70637380-3e90-11ea-8b0b-22df0a4c9dba.jpg)

**Android OS version:**
9

**Device manufacturer and model:**
Samsung Galaxy 9

**List of installed add-ons (like languages, or themes):**
German Language Pack

**Comment by Slion** [comment]
You can already change the layout of the bottom row from the UI settings.

**Comment by Glitchy-Tozier** [comment]
@Slion The backspace is not part of the bottom row.

**Comment by nicoursi** [comment]
This issue is German specific. If you check, eg. `English`, the back space is bigger. It can be done. The question is about creating a new layout or modifying the `German Qwerty slim`, where there is enough space. And also creating  a `German QWERTZ slim` with a bigger backspace. `German neo 2 simple` already has a bigger backspace. If you can't find these layouts in your keyboard, subcribe to the beta releases or wait for the next stable.

**Comment by Glitchy-Tozier** [comment]
You're right! I never noticed that backspace had a proper size in English layouts. Thank you for pointing that out!

As for the implementation, I think #2011 might be a good start... I'm not sure, however, whether it's better to add a new layout or have a checkbox somewhere. A completely separate layout might be more intuitive for the user though.
